WordPress/src/main/java/org/wordpress/android/datasets/PeopleTable.java

```diff
@@ -26,6 +25,7 @@ private static SQLiteDatabase getWritableDb() {
     public static void createTables(SQLiteDatabase db) {
         db.execSQL("CREATE TABLE " + PEOPLE_TABLE + " ("
                 + "person_id               INTEGER DEFAULT 0,"
+                + "site_id                 TEXT,"
```

Comment:
I might be wrong here as well, but I think it's actually the opposite. I know there is a push to use `site` instead of `blog`, especially on Calypso, and if you take a look at the latest additions like `SiteSettings`, `DeleteSite` etc, we are doing something similar on Android as well. I do agree though that `local_blog_id` would be clear if we used `blog_id`. The reason I didn't use `local_site_id` there is, because we use `local_blog_id` everywhere else. I feel like `site_id` here makes more sense just because the only use of it is for the network calls. Again, I am really not sure.

@maxme Would you like to weigh in here on way or the other?